src/dvc_render/vega_templates.py

```diff
@@ -598,6 +598,123 @@ class LinearTemplate(SmoothLinearTemplate):
     DEFAULT_NAME = "linear"
 
 
+class LogLinearTemplate(Template):
+    DEFAULT_NAME = "log_linear"
+    DEFAULT_CONTENT = {
```

Comment:
minor: I think there might be a way to inhering from `LinearTemplate` to not have to define the entire default content again